### PR TITLE
Feature/organizational page

### PR DESCRIPTION
**Pull request by jmgayosso**
I developed all organizational widgets, and fixed some issues with size paddings and no-wraps titles or descriptions

### CODE REVIEW

File: .env.example [hunk 1]
```diff
@@ -21,3 +21,7 @@ DGRAPH_URL=http://ec2-3-236-210-156.compute-1.amazonaws.com:8080
 TELOS_ENDPOINTS=https://testnet.telos.caleos.io
 
 TLOSTO_SEEDS=tlosto.seeds
+
+IPFS_URL=ipfs.infura.io
+IPFS_PROJECT_ID=20mUXy1Rao9DE32ei9mWy9TlpYM
+IPFS_PROJECT_SECRET=3a2b32602886aca0389a2b6294cf65a4
```
Comment (lukegravitt) [inline .env.example:27]: Probably don't want this committed... but it's too late now. Probably safest bet is to change this SECRET key and then you can leave this one here (once its essentially the wrong key)

File: .env.example [hunk 2]
```diff
@@ -21,3 +21,7 @@ DGRAPH_URL=http://ec2-3-236-210-156.compute-1.amazonaws.com:8080
 TELOS_ENDPOINTS=https://testnet.telos.caleos.io
 
 TLOSTO_SEEDS=tlosto.seeds
+
```
Comment (lukegravitt) [inline .env.example:24]: If you need to add github secrets, see the configuration yamls for examples.

File: src/components/common/widget.vue [hunk 3]
```diff
@@ -48,7 +48,7 @@ q-card.widget(flat :class="{ 'shadowed': shadow }")
   border-radius 32px
 
   .shadowed
-    box-shadow 0 4px 8px rgba(0 0 0 0.05), 0 1px 16px rgba(0 0 0 0.025) !important
+    box-shadoqw 0 4px 8px rgba(0 0 0 0.05), 0 1px 16px rgba(0 0 0 0.025) !important
```
Comment (lukegravitt) [inline src/components/common/widget.vue:51]: Typo

File: src/components/ipfs/input-file-ipfs.vue [hunk 4]
```diff
@@ -0,0 +1,119 @@
...
+  },
+  methods: {
+    async downloadFile () {
+      const file = await BrowserIpfs.retrieve(this.typeCid)
+      const urlFile = URL.createObjectURL(file.payload)
+      window.open(urlFile, '_blank')
+    },
+    async updateModel (e) {
+      try {
+        console.log('updateModel', e)
```
Comment (lukegravitt) [inline src/components/ipfs/input-file-ipfs.vue:75]: remove the console.logs

File: src/components/login/welcome-view.vue [hunk 5]
```diff
@@ -0,0 +1,41 @@
...
+    q-btn.full-width.q-mt-sm(
+      label="Login"
+      color="secondary"
+      rounded
+      no-caps
+      @click="$emit('onLoginClick')"
+    )
+    //- to="/preview"
+    q-btn.full-width.q-mt-sm(
+      to="/preview"
```
Comment (lukegravitt) [inline src/components/login/welcome-view.vue:29]: Use a named path here (and everywhere) -- :to="{ name:'dashboard' }"

**Review by lukegravitt — COMMENTED** [review]
I think you are on to right track here. I'm sorry I wasn't able to dig as deeply as needed just due to time constraints. Looks good.

You will need to make adjustments here to merge on top of my code which I'm putting on develop soon.

File: .env.example (hunk 1 above)
Comment (jmgayosso) [reply .env.example:27]: I got it, I will take a look at it

File: src/components/login/welcome-view.vue (hunk 5 above)
Comment (jmgayosso) [reply src/components/login/welcome-view.vue:29]: done

File: src/components/ipfs/input-file-ipfs.vue (hunk 4 above)
Comment (jmgayosso) [reply src/components/ipfs/input-file-ipfs.vue:75]: done

File: src/components/common/widget.vue (hunk 3 above)
Comment (jmgayosso) [reply src/components/common/widget.vue:51]: fixed

**Pull request merged**